React/exercises/identifying-components.md

```diff
@@ -1,14 +1,42 @@
 # Identifying Components
+
+## Part One: Do you Agree or Disagree
 In this exercise, we will go through a popular website and brainstorm how to break down the web application into components.
 
-To get you started, we've done an example of this using the MediaRanker application that you're already familiar with.
+To get you started, we've done an example of this using a MediaRanker application.
+
+Look at this screenshot of a MediaRanker application (the site in black and white). We've outline the components in colorful boxes and titles.
 
 ![Media Ranker as Components](../images/media-ranker-components.png)
 <!-- https://docs.google.com/drawings/d/1zWLaZfPatvVE7cFS2qdpnOLgsyW45ZJP1YDEGWECi7g/edit -->
 
-Break into pairs and take a look at CraigsList. Identify the visual "components" the website is comprised of. We suggest using using your white board or paper to draw these out on your table!
+Our list of proposed components from this website is:
+
+- Navigation
+- MediaSpotlight
+- TopMedia
+- Media
+
+Answer the following questions:
+
+- For each component, what is its name, what does it represent?
+- For each component, what website elements are inside of it?
+- For each component, is it a sibling to another component?
+- For each component, are there another components nested inside of it?
+- For each component, does it get used more than once in this page? If yes, what are the differences in UI between each time it's used?
+- Do you agree with this list of components, what they represent, and how they're named? Do you have any disagreements, or other ideas?
+
+## Part Two: Make a List of Possible Components with CraigsList
+
+Break into pairs and take a look at [CraigsList](https://seattle.craigslist.org/).
```

Comment:
Are we planning to facilitate the breaking into pairs? If not, but we do want them to talk with someone about it somehow, I think it'd be helpful to give more direction for C13. 
But also maybe you were planning to leave this as if for future cohorts and communicate about this to students separately. Totally open to that :)

Reply:
My intention was to leave the wording vague in order for this to be useful for C14, but I'll just go ahead and update this now to just "Take a look" without the pairs. Thank you :+1: :smile:
6